docs/docs/api_reference.md

```diff
@@ -1,6 +1,8 @@
 ---
 slug: /api/
 sidebar_position: 4
+toc_min_heading_level: 2
```

Comment:
If other documents also exceed two levels in the future, this can be modified as a global configuration.